package.json

```diff
@@ -4,45 +4,34 @@
   "description": "A starter kit for building PWAs!",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "rimraf dist/ && rollup --config rollup.config.js && node workbox.mjs",
-    "start": "concurrently \"rollup --config rollup.config.dev.js --watch\" \"web-dev-server --app-index build/index.html --root-dir build/ --compatibility none --node-resolve --watch --open /\"",
-    "dev": "concurrently \"rollup --config rollup.config.dev.js --watch\" \"web-dev-server --app-index build/index.html --root-dir build/ --compatibility none --node-resolve --watch --open /\"",
-    "dev-task": "concurrently \"rollup --config rollup.config.dev.js --watch\" \"web-dev-server --app-index build/index.html --root-dir build/ --compatibility none --node-resolve --watch /\""
+    "dev-task": "vite",
```

Comment:
consider calling this 'dev' instead for folks who are used to 'npm run dev'

Reply:
ahh, I think this was actually changed as a mixup, good catch